packages/postgres/README.md

```diff
@@ -1,4 +1,6 @@
-# @vercel/postgres
+# @vercel/postgres (deprecated)
+
+> [!CAUTION] > **`@vercel/postgres` is deprecated**. Please use [the `@neondatabase/serverless` package from Neon instead](https://github.com/neondatabase/serverless). See the [Vercel Postgres to Neon transition guide](https://neon.tech/docs/guides/vercel-postgres-transition-guide).
```

Comment:
do we need to deprecate https://github.com/vercel/storage/tree/main/packages/postgres-kysely as well?

Reply:
yep, apologies! I think I pushed an older commit.